% LaTeX template for (Academic) CV.
% The template consists only of this file.
% Document is to be structured as a normal article.
% Created in 2017 by Pedro Tiago Martins (ptmartins.info).
% Dedicated to the public domain.

\documentclass[12pt]{article}
\usepackage{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{enumitem}
\usepackage{libertine} %font
\usepackage[margin=1.7cm]{geometry} %margins
\addtolength{\topmargin}{-.25in}
%\addtolength{\textheight}{1.75in}
\usepackage{fancyhdr} % header/footer
\usepackage{lastpage} %last page (for footer)
\usepackage[hidelinks]{hyperref}
\pagestyle{fancy}
\fancyhead{}
\fancyfoot{}
\renewcommand{\headrulewidth}{0pt}
\fancyfoot[R]{\thepage/\pageref{LastPage}}

\usepackage[hidelinks]{hyperref}
\hypersetup{
  colorlinks,
  citecolor=Violet,
  linkcolor=Black,
  urlcolor=Blue
  }
\usepackage[usenames,dvipsnames,svgnames,x11names,table]{xcolor} % color
\newcommand{\cvcolor}{\color{DarkRed}} %easier to change color consistently
\usepackage{doi} % easy input of doi numbers
\usepackage{enumitem} % customized lists (reset counters with the resume option)
\usepackage{titlesec} % customized section headers
\usepackage{titling} % customized title
\setlength\parindent{0em} 
\setlength{\parskip}{0.5em}

%Section Header format
\titleformat{\section}
{\LARGE\scshape}{}{0em}{\cvcolor}[\titlerule]

\titleformat{\subsection}
{\large\scshape}{}{0em}{\cvcolor}

\titleformat{\subsubsection}
{\bfseries}{}{0em}{}

%Section header spacing
\titlespacing{\section}
{0em}{0em}{0em}

\titlespacing{\subsection}
{0em}{0.75em}{0em}

\titlespacing{\subsubsection}
{0em}{0em}{0em}

%Title

\renewcommand{\maketitle}{
\begin{center}

{\Huge\theauthor}\\
\vspace{0.25em}
{\Large \cvcolor{Stanford University, California}}\\

\href{mailto:nikgvetr@stanford.edu}{nikgvetr@stanford.edu} --- \href{https://www.github.com/NikVetr/}{github.com/NikVetr/} --- \href{https://www.linkedin.com/in/nikolai-vetr}{linkedin.com/in/nikolai-vetr}\\

{\color{Gray}(last update: \today)}
\end{center}
}

\begin{document}

\title{Curriculum Vitae} % doesn't actually show up anywhere
\author{Nikolai Vetr}

\maketitle

\vspace{-1.5em}

\section{Education and Training}

\textbf{Postdoc}, Montgomery Lab, Stanford University\hfill\emph{Current}\\
Pathology + Biomedical Data Science
\vspace{-0.25em}

\textbf{PhD}, University of California, Davis\hfill\emph{2020}\\
Dissertation: \textit{Exploring and Extending Multivariate Brownian Diffusion Models of Phenotypic\\
\hspace*{22mm} Evolution for Bayesian Phylogenetic Inference}\\
Committee: Tim Weaver (Chair), Shara Bailey, Peter Wainwright\\
GPA: 4.0/4.0 — Anthropology + Population Biology + Data Science \& Informatics

\textbf{BA}, Vanderbilt University\hfill\emph{2013}\\
GPA: 3.9/4.0 — Earth \& Environmental Sciences + Ecology, Evolution \& Organismal Biology\\
Departmental Honors, \textit{summa cum laude}\\
\vspace{-1.25em}
\subsection{Additional Education}

\textbf{Discussion Groups}\hfill\emph{2013 - 2020}\\
I've served as an active participant in groups on Computational Molecular Evolution, Paleoanthropology, Quantitative Genetics, and Machine Learning. I've been the founder and lead in groups on Python, Applied Linear Algebra, Bayesian Data Analysis, and Deep Learning.

\textbf{Workshops}\hfill\emph{2013 - 2020}\\
I've attended or led multiple workshops ranging in duration from 2h to 9d on Universal Design, Science Communication, Bayesian Phylogenetics, Database Management, Natural Language Processing, Carpentries Instructor Training, and other topics.

\section{Languages}

\noindent\begin{tabular}{@{}ll}
\hspace{2cm} \textbf{Programming}         & \hspace{3cm}                          \textbf{Natural}\\
\textbf{Fluent:} & \hspace{3cm} Russian, English\\
\textbf{Proficient:} R, RevBayes, Stan, BASH &  \hspace{3cm} Spanish\\
\textbf{Beginner:}  \hspace{0.035cm} Python, C\texttt{++}, CSS, HTML, JS &  \hspace{3cm} \\
\end{tabular}


\section{Service}

\subsection{Journal Review}

\emph{Evolution} (2017),  \emph{Science Communications} (2018),  \emph{Cell Reports} (2021), \emph{Human Genetics and Genomics Advances} (2022)

\subsection{Grant Review}

\emph{WAI Grants} (2021,  2022,  2023)

\section{Leadership}

\textbf{Founder \& Lead}, Applied Bayesian Statistics Research Cluster \hfill \emph{2019 - 2020}\\
I founded, coordinated, and supervised an interdisciplinary research cluster of 70+ scientists and statisticians across numerous career stages (PhD Student, Postdoc, PI, Industry Researcher), financially and spiritually sponsored by the UC-Davis \emph{Data Science \& Informatics} Unit\\\\
\textbf{Director \& President}, Board of Directors, Wild Animal Initiative \hfill \emph{2020-Present}\\
I serve on the Board of Directors at the \textit{\href{https://www.wildanimalinitiative.org/}{Wild Animal Initiative}}, a non-profit dedicated to researching the lives of wild animals. In this role, I help to oversee the organization's activities and policies as described in their mission statement and bylaws, approve financial decisions, decide on board meeting agendas, and engage in related activities.  I've also served as chair of both the DEI and Science committees\\\\
\textbf{Director \& President}, Board of Directors, Rethink Priorities \hfill \emph{2023-Present}\\
I serve on the Board of Directors at \textit{\href{https://rethinkpriorities.org/}{Rethink Priorities}}, a non-profit focused on cause prioritization and research in animal welfare, artificial intelligence, climate change,  and global health and development. Responsibilities are similar to above. Over several years I have also extensively consulted on or performed statistical model fitting, data visualization, and related activities
%\textbf{Coordinator},  Reading \& Discussion Groups \hfill \emph{2014 - 2020}\\
%I've founded and coordinated reading groups on Python, Applied Linear Algebra, Bayesian Data Analysis, and Deep Learning\\
%\textbf{Undergraduate Student Mentor}, UC-Davis \hfill \emph{2014 - 2020}\\
%While serving as a college instructor, I've mentored several undergraduate students, meeting frequently to advise them on their career and research aspirations

\section{Projects}

\subsection{Postdoctoral Work}

My postdoctoral work in the Montgomery Lab focuses on exploring and developing statistical methods to better understand the molecular mechanisms by which genetic and environmental variation structure phenotypic variation, especially insofar as that understanding might have translational applications in contexts such as drug discovery and personalized medicine.  Current and recent projects include:

\begin{itemize}[noitemsep]

\item Integrating GWAS and eQTL mapping results with transcriptomic, proteomic, and ATAC-seq data to help \textbf{identify the molecular transducers of physical activity} through the MoTrPAC Consortium [\href{https://github.com/NikVetr/MoTrPAC\_Complex\_Traits}{LINK}]

\item Understanding genomic influences on \textbf{allele-specific expression in the context of ovarian cancer} through the course of tumor development [\href{https://github.com/nsabell/egtex-ase}{LINK}]

\item Developing \textbf{integrative methods to link multiomic response with trait expression} that appropriately propagate upstream inferential uncertainty [\href{https://github.com/NikVetr/papers/blob/main/twas-method/proposed\_TWAS\_method.pdf}{LINK}]

\item Hiearchically modeling the \textbf{temporal dynamics of whole-trancriptome expression data} following environmental perturbation using linear combinations of logistic functions to characterize temporal behavior in expected rates of gamma-poisson distributed read counts [\href{https://github.com/NikVetr/montgomery_lab/blob/61e2ac4b0fea0ee7f5bf31590a40378f0027264a/growth\_and\_decay\_processes.R#L8}{LINK}]

\item Exploring causal inferential implications of \textbf{aging clocks} [\href{https://github.com/NikVetr/montgomery\_lab/blob/61e2ac4b0fea0ee7f5bf31590a40378f0027264a/collider\_effects\_on\_R2.R#L4}{LINK}, \href{https://drive.google.com/file/d/1AQU0YMVoYCEFuIICHGIXGxoXm6dj7HxY/view}{example}]

\item Developing easy-to-use tools for \textbf{frequentist testing} of novel generative immune response models

\item Describing \textbf{novel informative correlation matrix distributions} for use in multivariate,  multilevel diffusion models [\href{https://github.com/NikVetr/dissertation\_work/blob/master/informative\_corrmat\_prior.R}{LINK}]

\item Understanding the proteomic signatures of \textbf{allogeneic renal transplant rejection} [\href{https://github.com/NikVetr/minor\_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/renal\_transplant\_censored\_poisson.R}{LINK}]

\item Exploring sources of linkage disequilibrium unrelated to crossing over (eg assortative mating) in a simulation framework [\href{https://github.com/NikVetr/montgomery\_lab/blob/61e2ac4b0fea0ee7f5bf31590a40378f0027264a/assortative\_mating.R}{LINK}]

\item Collaborating on the description and implementation of \textbf{Bayesian differential expression models} \href{https://github.com/bob-carpenter/BayesExpress/blob/main/latex/bayes-express/bayes-express.tex}{[LINK]},  

\item Describing theoretical expectations for \textbf{rare-variant effects on outlier detection} [\href{https://github.com/NikVetr/montgomery_lab/blob/61e2ac4b0fea0ee7f5bf31590a40378f0027264a/prop\_outliers\_zscores\_animation.R#L4}{LINK}, \href{https://drive.google.com/file/d/1mSp22\_cqarfMDFnS55Pjg7qqmkVnk1GI}{example}]

\end{itemize}

\subsection{Dissertation Work}

My dissertation work primarily centered on Bayesian computational phylogenetics methods development, specifically extending and exploring multivariate Brownian (mvBM) diffusion models to novel character evolutionary applications and contexts [\href{https://github.com/NikVetr/papers/blob/main/dissertation/dissertation_lashinsky_2020.pdf}{LINK}]. Components of this this work included:

\begin{itemize}[noitemsep]

\item through simulation \textbf{exploring the informativeness and reliability of continuous character alignments} at retrieval of simulating tree topology, with subsequent application to a craniometric dataset 
\item the \textbf{effects of model misspecification} in the above, both with respect to the evolutionary process and with respect to accommodating missing data and measurement error
\item developing and implementing novel optimizations to \textbf{reduce the time complexity} of likelihood calculation under the phylogenetic mvBM model from O($n^3$) to O($n^2$), modularizing particular operations to \textbf{avoid redundant computation} during Met-Hastings, and inventing substantially \textbf{more efficient, tunable proposal distributions} over correlation matrices of arbitrary dimensionality while inherently respecting PSD constraints. In total, \textbf{reducing the runtime required} to fit this popular evolutionary model under typical empirical conditions from months to hours
\item \textbf{comparing model-based phylogenetic methods to more common heuristic (maximum parsimony, distance) methods}, both in simulation and empirically (using ape and old-world monkey datasets), and contrasting their performance to methods that use molecular (e.g. nucleotide sequence) data
\item \textbf{joint inference of ancestral character states} in a monkey dataset, with additional theory development on imputing unobserved tip characters in a phylogenetically sensible way 
\item layering a \textbf{high-dimensional probit model atop mvBM to accommodate discrete traits} and comparing it with both more traditional and novel CTMC (\emph{Continuous Time Markov Chain}) models of discrete trait substitution, with application to human dental variation

\end{itemize}

In addition to the above dissertation contents, parallel work involving related skills included:

\begin{itemize}[noitemsep]

\item \textbf{comparison of morphological relaxed clock models} in a species-rich fish dataset whose tree height straddles the K-Pg boundary [\href{https://github.com/NikVetr/dissertation_work/blob/961baf0f64fe6139dc2c1723d37dcd4d51cbd16a/rateHeterogeneity\_DisparityThroughTime.R}{LINK}]
\item \textbf{approximating truncated biogeographic diffusions using data augmentation} on phylogenies to sample migration histories while respecting (land, water) barriers to explore latitudinal / climactic drivers of morphological differentiation in modern humans [\href{https://github.com/NikVetr/dissertation\_work/blob/961baf0f64fe6139dc2c1723d37dcd4d51cbd16a/truncatedBMapprox.R}{LINK}]
\item high-dimensional multilevel generalized linear mixed models (GLMMs; Gamma-Poisson, in this case) in Stan to \textbf{heart transplant patient immune response} data both before and after an acute rejection episode [\href{https://github.com/NikVetr/side\_projects/blob/master/multilevelPoisson.R}{LINK}]
\item a hierarchical univariate Ornstein-Uhlenbeck model in Stan to the `evolution' of \textbf{nitrogen concentrations in manure ponds} across California to predict values after an arbitrary amount of time has passed in an arbitrary pond [\href{https://github.com/NikVetr/side_projects/blob/24f7e38876a95433364123441f9b41d260105be6/OUmodel\_manurePonds.R}{LINK}]

\end{itemize}

%
%[\href{}{LINK}, \href{}{example}]
%[\href{}{LINK}, \href{}{example}]
%[\href{}{LINK}, \href{}{example}]
%[\href{}{LINK}, \href{}{example}]

\subsection{Personal Projects}
I've written many dozens of one-off scripts devoted to data visualization, text mining, web scraping, animation, replicating published analyses or algorithms from scratch, generating artwork, solving everyday diffuclties, and exploring off-the-cuff ideas proposed by myself or colleagues.

Recent examples here include but are not limited to:

\begin{itemize}[noitemsep]

\item Leveraging wavelets and other transforms to \textbf{compress and vectorize raster images} [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/vectorize\_raster\_image.R}{LINK}, \href{https://drive.google.com/file/d/1rrEjdL13qZZIi22Pe5IkbUJKjPmpPpP5}{example}], with initial attempts to expand into higher dimensional spaces (eg time or 3D objects) [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/interpolate\_signature\_frequency\_series.R}{LINK}, \href{https://drive.google.com/file/d/1IDgm04h0LEZc4IXOPjqBheifsz2Y6ZmW}{example}]

\item Automated \textbf{infilling and text-wrapping} of arbitrary images with arbitrary text [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/text\_to\_image.R}{LINK}, \href{https://drive.google.com/file/d/1cedbUpq8ArDd4fnjxUZBHlSLuVqsI7UO}{example}]

\item Automated \textbf{processing and overlay of in/outpainted images} [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/dall-e2\_zoomout.R}{LINK}, \href{https://drive.google.com/file/d/1nuvGY8d3h9qYeh9NrWVbo9ybkt0RdORn}{example}]

\item \textbf{Generating seating charts} under various (attitudinal, geographic) optimization criteria for an icebreaker activity, including in the multi-round iterated case where participants move [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/optimal_table_seating.R}{LINK}, \href{https://drive.google.com/file/d/1z-D3m9fd-hYQmAHGoDBl3N--k-udfTLb/view?usp=drive\_link}{example}]

\item Making other \textbf{small animations} leveraging basic geometric and linear algebraical operations [\href{https://github.com/NikVetr/minor_scripts/blob/c8926965b0c6dd9158351cf5836cbc76261f6830/postdoc/polygon\_animation.r}{LINK}, \href{https://drive.google.com/file/d/1BCUmNXSr2ST00JxM32t7qBhQXMZEZERv}{example}]

\item Fitting multilevel GLMMs in Stan to \textbf{consumer dietary and attitudinal response datasets} (zero-inflated Poisson and ordered logit, respectively) measuring the effects of exposure to a relevant advertisement urging dietary change [\href{https://github.com/NikVetr/side\_projects/blob/24f7e38876a95433364123441f9b41d260105be6/MFA_Ads.R}{LINK}]

\item Implementing an efficient, conditional multivariate ordinal probit model for \textbf{personalized movie rating predictions}, exploiting basic properties of Schur complements [\href{https://github.com/NikVetr/side\_projects/blob/24f7e38876a95433364123441f9b41d260105be6/movieRatingPersonalization.R}{LINK}]

\item Writing a \textbf{``falling rain'' parody generator} for the Matrix 4 release [\href{https://github.com/NikVetr/montgomery\_lab/blob/61e2ac4b0fea0ee7f5bf31590a40378f0027264a/Matrix\_GreenRain.R#L4}{LINK},  \href{https://drive.google.com/file/d/1hwzzLq9xTpAc2AvFyLTy-L\_ZHG\_VlIK\_}{example}]


\end{itemize}

I've also consulted individuals \& organizations in a minor role (e.g. over a $\sim 1$h meeting) on many additional projects throughout my postdoc and graduate school.

\subsection{Field \& Labwork}

\textbf{Archaeological / Paleontological Excavation} \hfill \emph{2013}\\
During the Summer 2013 field season I helped in excavation efforts ongoing at La Ferrassie, a Neandertal archaeological and paleontological site in Savignac-de-Miremont, France

\textbf{Water Quality Analysis} \hfill \emph{2011}\\
I performed water quality analyses (of salinity, reactive nitrogen concentration, etc.) of a large set of streams distributed across the North Island of New Zealand

\textbf{Stable Isotope Ecology} \hfill \emph{2012 - 2013}\\
I examined how environmental and ecological conditions were recorded in enamel stable isotope ratios (carbon and oxygen) and dental microwear textures for several marsupial taxa throughout Australasia

\section{Teaching}

\textbf{Associate Instructor$^{*}$}, University of California, Davis \hfill \emph{2015  - 2020}\\
As instructor-of-record, I’ve taught three quarters of an upper-division paleoanthropology course, two quarters of an upper-division evolutionary primatology course, and one quarter of an introductory course on human evolution. In these roles, I created or modified all the lab and lecture materials, designed and graded assignments and tests, mentored individual students, delivered lectures, and either supervised teaching assistants or administered and oversaw lab activities

\textbf{Teaching Assistant$^{*}$}, University of California, Davis \& Vanderbilt University \hfill \emph{2012  - 2020}\\
I have taught a total of 14 discussion sections for courses on human evolution and paleoanthropology, as well as the lab for an introductory cell biology course at Vanderbilt University. As a TA, I occasionally gave guest lectures on various subjects (mostly statistical phylogenetics)

\textbf{Outreach Lecturer}, University of California, Davis \hfill \emph{2013  - 2020}\\
I've given multiple yearly talks on human evolution to elementary and middle school students on location and visiting UC-Davis, as well as during campus-wide events (e.g. \emph{Picnic Day}). I've also assisted in miscellaneous workshops targetted at adults, e.g. on NLP during UC-D Data Science Health Day

\textbf{Course Coordinator}, Workshop in Applied Phylogenetics \hfill \emph{2019}\\
I served as a course coordinator for world-renowned, widely-attended, week-long workshop in applied computational Bayesian phylogenetics held in May 2019 at the Bodega Marine Laboratory

\textbf{Carpentries Instructor}\hfill \emph{2019}\\
I have completed Instructor Training and Checkout for The Carpentries organization and am fully certified to teach courses in the Data and Software Carpentries

$^{*}$\emph{student evaluations available upon request}

\section{Research Output}

\subsection{Papers}

\begin{enumerate}[label={[\arabic*]}]



\item Abell, N., \textbf{Vetr, N.*},  Montgomery, S., et al.  2023.  \emph{A Survey of High Depth Allele-Specific Expression Across Normal Tissues and Ovarian Cancers}.  In Prep.

\item \textbf{Vetr, N.},  Gay,  N.,  and Montgomery,  S.  2023.  \emph{The impact of exercise on gene regulation in association with complex trait genetics}.  Conditionally accepted to Nature Communications pending Editorial Revisions. [ \href{https://github.com/NikVetr/MoTrPAC_Complex_Traits}{LINK}]

\item \textbf{MoTrPAC Study Group}$^{\dag}$. 2022.  \emph{Temporal dynamics of the multi-omic response to endurance exercise training across tissues}.  Accepted to Nature.  [\href{https://www.biorxiv.org/content/10.1101/2022.09.21.508770v2}{LINK}]

\item  Carpenter,  B.,  Chen,  S.,  and \textbf{Vetr,  N}.  2022.  \emph{Bayesian Models of Gene Expression and RNA Sequencing}.  \href{https://github.com/bob-carpenter/BayesExpress/blob/main/latex/bayes-express/bayes-express.tex}{[LINK]}

\item Gates, K., Panicker, A., Biendarra-Tiegs, S., \textbf{Vetr, N.}, Higuita, M., Nelson, T., Pereira, N., and Griffiths, L. 2021.  \emph{Shotgun Immunoproteomics for Identification of Nonhuman Leukocyte Antigens Associated With Cellular Dysfunction in Heart Transplant Rejection}. Transplantation 106(7). DOI: 10.1097/TP.0000\\000000004012 [\href{https://pubmed.ncbi.nlm.nih.gov/34923540/}{LINK}]

\item \textbf{Vetr, N.}, Bailey, S., Moore, B., and Weaver, T. 2020.  \emph{Human Population History from Discrete Dental Traits Under an Approximate Multivariate Ordinal Probit}. [\href{https://github.com/NikVetr/papers/blob/main/plosone-manuscript/mvBM_manuscript_plosone.pdf}{LINK}]

\item \textbf{Vetr, N.} and Weaver, T. 2020.  \emph{Primate Phylogenetics with Landmark Data: A Model-Based, Multivariate Brownian Approach}. [\href{https://github.com/NikVetr/papers/blob/main/jhe-manuscript/mvBM_manuscript_JHE.pdf}{LINK}]

\item \textbf{Vetr, N.}, May, M., Moore, B., and Weaver, T. 2019.  \emph{Bayesian Phylogenetic Inference Under Multivariate Brownian Motion}. [\href{https://github.com/NikVetr/papers/blob/main/sysbio-manuscript\%20/mvBM_manuscript_sysbio.pdf}{LINK}]

\item \textbf{Vetr, N.}, DeSantis, L., Yann, L., et al. 2013.  \emph{Is Rapoport’s Rule a Recent Phenomenon? A Deep Time Perspective on Potential Causal Mechanisms}. Biology Letters 9(5): 1-5. DOI: 10.1098/rsbl.2013.0398 [\href{https://royalsocietypublishing.org/doi/10.1098/rsbl.2013.0398}{LINK}]

*\textit{dual first authorship}\\
$^{\dag}$\textit{Author Group: 2 (of 8)}

\end{enumerate}



\subsection{Conference Presentations}

\begin{enumerate}[label={[\arabic*]}]

\item \textbf{Vetr, N}., Gay, N., and S. Montgomery. 2022. \emph{The impact of exercise on gene regulation in association with complex trait genetics}. Poster session presented at the American Society of Human Genetics' annual meeting, 2022.

\item \textbf{Vetr, N}., Gay, N., and S. Montgomery. 2022. \emph{Integrating Exercise-Induced Gene Expression and Human Phenotypic Variation}. Poster session presented at NIH-NLM T15 Training Conference 2022.

\item \textbf{Vetr, N}., May, M. Moore, B., and T. Weaver. 2018. \emph{Adapting multivariate Brownian diffusion models to Bayesian inference of human population history and phylogeny}. Oral session presented at the 87th Annual Meeting of the American Association of Physical Anthropologists.

\item Desantis, L., \textbf{Lashinsky, N}., Romer, J., Greshko, M., and Loffredo, L. 2013. \emph{University Students' Acceptance of Climate Change and Evolution: Are Skeptics just Anti- Science?} Poster presented despite second authorship at the 73rd Annual Meeting of the Society of Vertebrate Paleontology. 

\item \textbf{Lashinsky, N}., DeSantis, L. Yann, S. Donohue, and R. Haupt. 2013. \emph{Is Rapoport’s Rule a Recent Phenomenon? A Deep Time Perspective on Potential Causal Mechanisms}. Oral session presented at the 73rd Annual Meeting of the Society of Vertebrate Paleontology. 

\item \textbf{Lashinsky, N}. and DeSantis, L. 2012. \emph{Assessing how the extant lesser forest-wallaby Dorcopsulus vanheurni and Lumholtz’s tree kangaroo Dendrolagus lumholtzi record their environment: establishing a baseline for studies of extinct marsupials}. Poster presented at the 5th Annual Meeting of the Southeastern Association of Vertebrate Paleontology. 


\end{enumerate}

\vspace{-0.35em}

\section{Grants \& Awards}

\textbf{NIH T15}\hfill \emph{2021}\\
\textbf{Excellence in Data Science Community Training and Outreach}\hfill \emph{2019, 2020}\\
\textbf{Outstanding Graduate Student Teaching Award Nominee}\hfill \emph{2016, 2019, 2020}\\
\textbf{Conference Travel Award}\hfill \emph{2012, 2018}\\
\textbf{1st Place Picnic Day Exhibit Award in “Secrets of Nature” Category}\hfill \emph{2017}\\
\textbf{NSF Graduate Research Fellowship}\hfill \emph{2015}\\
\textbf{Summer Research Grant}\hfill \emph{2014, 2015}\\
\textbf{Graduate Scholars Fellowship}\hfill \emph{2013}\\
\textbf{Departmental Honors in Earth and Environmental Sciences}\hfill \emph{2013}\\
\textbf{Eugene H. Vaughan Undergraduate Research Assistantship in Geology }\hfill \emph{2012}\\
\textbf{Geology Travel Grant}\hfill \emph{2012}\\
\textbf{Vanderbilt Undergraduate Summer Research Grant}\hfill \emph{2012}\\
\textbf{Ross Family Scholarship}\hfill \emph{2012}\\
\textbf{National Merit Scholarship}\hfill \emph{2009}\\

%
%\vspace{-0.35em}

%\section{Miscellaneous}
%\textbf{Computer Hardware}\\For my dissertation work, I constructed and networked an 8-node (64C/128T) Ryzen-based Beowulf cluster using \emph{GNU Parallel}, as well as a portable 5-node Raspberry Pi cluster running Linux for teaching. I have designed and assembled similar systems for academic colleagues\\\\
%\textbf{Outdoors}\\ I have coordinated and led many hiking, backpacking, and climbing trips of varying group size over distances ranging upwards to 1,300km, and have also backpacked and hitchhiked through several countries. I also sporadically partake in other adventure sports\\\\
%\textbf{Photography}\\ I am a hobbyist photographer, graphic designer, and illustrator focusing on portrait, landscape, and wildflife photography, with post-processing typically done in Capture One and Adobe Photoshop and recently augmented with more sophisticated computational tools (e.g. ANNs)\\

%Other hobbies include woodworking, strength training, running, yoga, writing poetry and prose, travel, and volunteering.
%\textbf{Carpentry}\\ I enjoy building things out of wood using both hand and power tools, even if I'm not very good at it.\\\\
%\textbf{Charity}\\ Active participant in the \emph{Effective Altruism} movement since it's inception; donate regularly to meta-charity approved organizations and occasionally serve as a volunteer statistical consultant

%\textbf{Academic Metrics}: GRE Scores: 169 (V), 170 (Q), 5.5 (W); SAT Scores: 800 (M), 800 (V); Cumulative Undergraduate GPA: 3.91, Cumulative Graduate GPA: 4.0
\end{document}